### DISCUSSION TITLE
Input create file highlights

### DISCUSSION THREAD

**Discussion by GiuseppeMP** [nvim-tree/nvim-tree.lua#2458] [Q&A] [answered]
Hey guys, I'm using transparent theme, everything is fine but in NvimTree:

![image](https://github.com/folke/noice.nvim/assets/4944009/bace5b91-d079-4eb0-986a-221f674e9160)

Can anyone tell me what the highlight is here and where this black background is coming from?

![image](https://github.com/folke/noice.nvim/assets/4944009/babca1e4-83e6-4a0a-be89-0aaf0f6571f4)

[My config.](https://github.com/GiuseppeMP/my-nvim-config/blob/f853778ff5fc3685e2b550caec78c1417d57017d/lua/user/plugins/configs/noice.lua#L34)

Thx!

**Comment by Akmadan23** [comment, marked as answer]
This seems to be an issue relative to noice rather than nvim-tree, since it uses `vim.ui.input` which by default doesn't have any highlighting capability.

Edit: I've seen @folke's [comment](https://github.com/folke/noice.nvim/discussions/612#discussioncomment-7224000), where he seems pretty sure it's not a noice issue. If so I assume it's a nui issue. I would exclude nvim-tree from the equation for the reasons stated above.

**Reply by GiuseppeMP** [reply]
@Akmadan23 ty!

**Reply by folke** [reply]
Noice does not provide a ui for `vim.ui.input`.
Probably dressing.nvim or similar plugin

**Reply by GiuseppeMP** [reply]
@folke  @Akmadan23 ty so much!

It was dressing.nvim:

```lua
require('dressing').setup({
    input = {
        win_options = {
            winblend = 0,
        },
    },
})
```

fixed:

![image](https://github.com/nvim-tree/nvim-tree.lua/assets/4944009/781b44d5-c5a6-451f-9fa6-ea3533e21e34)
